### ISSUE TITLE
[A11y] Search Results - Grid - Image alt text

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1918]
### Operating System

Windows

### Browser

Edge

### Affected page

Search

### Admin user?

Yes

### Issue Description

Search result images are all missing the `alt` attribute

### Suggested Fix

Because the image and the link text are wrapped in the same `<a>` tag, `alt=""` would be the best choice here.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Inspect the page to ensure all images have an alt attribute, and that it makes sense in context.  In this context, images can be marked decorative because the text is part of the same link.
